.swiftlint.yml

```diff
@@ -3,6 +3,7 @@ disabled_rules: # rule identifiers turned on by default to exclude from running
   - trailing_comma
   - trailing_whitespace
   - sorted_imports
+  - private_over_fileprivate
```

Comment:
I think I found a pretty good use case for `fileprivate`: SwiftUI previews. There is no reason they ever need to be accessed outside their own file. Are you ok with disabling this rule @martindsq?

Reply:
👌 It is a good rule to have in mind, but not to enforce it, as in this case